README-uk.md

```diff
@@ -333,7 +333,7 @@
 ## Інші джерела
 
 - [awesome-shell](https://github.com/alebcay/awesome-shell): Розширює Shell.
-- [awesome-osx-command-line](https://github.com/herrbischoff/awesome-osx-command-line): A more in-depth guide for the Mac OS command line.
+- [awesome-osx-command-line](https://github.com/herrbischoff/awesome-osx-command-line): Поглиблений гайд до командного рядка OS X.
```

Comment:
For the translations I think it is preferred to use native Ukrainian words instead of transliterated English, that do not exist in Ukrainian.

Could we use "підручник", "посібник", "керівництво" instead of "гайд"?